src/pages/Dashboards/dashboards/SocialTool/TrendsExploreInput/TrendsExploreInput.module.scss

```diff
@@ -0,0 +1,17 @@
+.wrapper {
+  max-width: 480px;
+}
+
+.input {
+  padding: 9px 13px;
+}
+
+.inputWithButton {
+  padding: 9px 94px 9px 13px;
+}
+
+.explore {
+  position: absolute;
+  right: 4px;
+  top: 4px;
+}
```

Comment:
Duplicate of the `src/components/Trends/TrendsExploreInput/TrendsExploreInput.module.scss`